.github/workflows/build.yml

```diff
@@ -32,14 +36,20 @@ jobs:
         architecture: 'x86'
     - name: Set up Python 3.8 x64
       uses: actions/setup-python@v2
-      if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
+      if: ${{ matrix.os == 'ubuntu-latest' }}
       with:
         python-version: 3.8
         architecture: 'x64'
+    - name: Set up Python for macOS
+      if: ${{ matrix.os == 'macos-latest' }}
+      run: |
```

Comment:
I'd add a comment explaining why we aren't using `actions/setup-python` here.

Reply:
fixed.